cmake/package.cmake

```diff
@@ -14,6 +14,14 @@ install(TARGETS
   PUBLIC_HEADER DESTINATION include
   FRAMEWORK DESTINATION ${CMAKE_INSTALL_BINDIR}
 )
+if(USE_CUDA)
```

Comment:
This is to pick up the `onnxruntime-genai-cuda.dll` after @RyanUnderhill 's work.